src/redux/categories/categories.js

```diff
@@ -1,17 +1,14 @@
 import { createAction } from '@reduxjs/toolkit/src/createAction';
+import { createReducer } from "@reduxjs/toolkit/src/createReducer";
 
+export const CHECK_STATUS = createAction('bookstore/categories/CHECK_STATUS');
```

Comment:
- [ ] This is an action creator as the previous reviewer said you should create actions separately as variables and use them in the actions creator like the following example 
```javascript
const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS'
and then use this variable inside the action creators 
```